core/lib/instrument_api.h

```diff
@@ -5052,6 +5052,7 @@ dr_insert_clean_call_ex(void *drcontext, instrlist_t *ilist, instr_t *where,
                         void *callee, dr_cleancall_save_t save_flags,
                         uint num_args, ...);
 
+DR_API
```

Comment:
Let's not add it to the API. It's probably not useful for clients and I can avoid it here.